cache/cache.h

```diff
@@ -0,0 +1,46 @@
+#ifndef CACHE_H
+#define CACHE_H
+
+/* A cached version of the global state database (object table, task log, task
+ * table). */
+typedef db_cache_impl db_cache;
+
+/* These are the error codes the methods can return. */
+#define CACHE_NOT_FOUND -1
+#define CACHE_OK 0
+
+/* Return an array of plasma managers that have the object (the last parameter
+ * is an out parameter, which is assumed to be an array we append to). */
+int cached_object_table_get(db_cache *db,
```

Comment:
Maybe change cached_* to cache_* ?

While for get() "cached" makes sense, it doesn't make that much sense for add().

In general the prefix of the function should be a noun as it is used to name the object on which the function operate on. Just emulate a poor man object oriented design ;-) 

object_name_method_name()

If we follow the above rule, maybe the best prefixes are object_cache, task_cache, etc.